renovate.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
This file must stay in the root directory.

Reply:
Okey